admin/templates/base.html

```diff
@@ -276,6 +278,7 @@
                   </ul>
               </div>
           {% endif %}
+          <li><a href="{% url 'internet_archive:internet_archive' %}"><i class='fa fa-link'></i> <span>Internet Archive</span></a></li>
```

Comment:
Should there be a permissions check here? Looks like best practice that we do that.

Reply:
Second this. Re-use change_node?